examples/native-external-references-mockshop/README.md

```diff
@@ -0,0 +1,312 @@
+
+> **NOTE:** 
+> Make sure you replace `<name-of-your-app>` with the name of your choice.
+
+```bash
+npx create-contentful-app@latest <name-of-your-app> --example native-external-references-mockshop
+```
+
+Once the process finishes, navigate to the directory that was created by running this command:
+
+```bash
+cd <name-of-your-app>
+```
+
+To complete the process, it is necessary to install all dependencies of the project by executing the following command:
+
+```bash
+npm i
+```
+
+## Creating a custom app definition
+
+To see your app within Contentful, you must first set it up. To do that, we will create an app definition, which is an entity that represents an app in Contentful and stores general information about it.
+
+To create the app definition, run this command:
+
+```bash
+npm run create-app-definition
+```
+
+You will need to answer the following questions in the terminal when prompted. Feel free to proceed with the default options provided.
+
+1. **Name of your application**. This is how your app will be named and it will be displayed in a few places throughout the UI. The default is the name of the folder you created.
+2. **Select where your app can be rendered**. This shows potential [app locations](https://www.contentful.com/developers/docs/extensibility/app-framework/locations/) where an app can be rendered within the Contentful Web app. Select **App configuration screen** , as we will utilize the configuration screen to provide the external API URL for the app.
+3. **Contentful CMA endpoint URL**. This refers to the URL used to interact with Contentful's Management APIs.
+4. **App Parameters**. These are configurable values that can be used to set default values or define custom validation rules. We need to define these for the API URL in this case:
+
+* Opt for **Y** to advance with the process of defining the parameters
+* Choose **Installation**
+* Input `MockShop API Endpoint` as **Parameter name** and `apiEndpoint` as **ID**
+* Select **Symbol** as type and mark the parameter as required
+
+5. The next steps will lead you through the process of providing a Contentful access token to the application and specifying the organization to which the application should be assigned.
+
+> **NOTE:** 
+> Make sure the organization ID you select here is of the organization that has access to the Native external references feature.
+
+You now have a basic application that can be enriched with additional information that will enable the example project you downloaded to function properly.
+
+## Building and uploading the app
+
+After creating the app definition, we can take care of uploading the code by running these commands:
+
+```bash
+npm run build && npx contentful-app-scripts upload --ci --bundle-dir ./build --organization-id <organisation-id> --definition-id <app-definition-id> --token <cma-token>`
```

Comment:
I'd split the commands instead of using `&&`. I would also recommend using the interactive cli for uploading.
```suggestion
npm run build
npm run upload
```

Reply:
I am fine with anything but this option seem really great all in one command but it is my opinion.